cita-cli/src/interactive.rs

```diff
@@ -424,3 +424,12 @@ impl GlobalConfig {
         println!("{}", output);
     }
 }
+
+fn remove_private(private_key: &str, blake_2b: bool) -> String {
+    let len = if blake_2b { 128 + 2 } else { 64 + 2 };
+    private_key
+        .split_whitespace()
```

Comment:
Please use `shell_words.split()`

What if the command is:
`rpc Xxx  --private-key "0xabcdef1230....."`